### DISCUSSION TITLE
How to enable/disable detailslist row selection checkbox?

### DISCUSSION THREAD

**Discussion by vechkar** [microsoft/fluentui#26499] [General]
I would like to enable/disable based on row status(one of the columns in the row) so that user will not select while performing any action. Please let me know if this can be done?

**Comment by ankitmaheshwarigithub** [comment]
You can use onRenderRow={onRenderRow}

 const onRenderRow = (props?: IDetailsRowProps): JSX.Element | null => {
    if (!props) {
      return null;
    }
    return (
      <div style={{ display: "flex", textAlign: "center" }}>
        <DetailsRow
          {...{
            ...props,
            disabled: !props.item.isPartOfOfferPackage,
            selectionMode: SelectionMode.multiple,
            selection: selection,
          }}
        />
      </div>
    );
  };
